ophyd/epics_motor.py

```diff
@@ -145,8 +145,18 @@ def move(self, position, wait=True, **kwargs):
             If motion fails other than timing out
         '''
         self._started_moving = False
+        status = None
```

Comment:
I don't think we need this line. Closures are allowed to refer to variables that are defined below them, as in:

```py
In [1]: def f(): 
   ...:     print(a) 
   ...:                                                                                                                                                                                       

In [2]: a = 1                                                                                                                                                                                 

In [3]: f()                                                                                                                                                                                   
1
```

Reply:
I agree.  Only added it to suppress warnings when logging was set to DEBUG.